src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -15,6 +15,8 @@ import ExonColumnFormatter from "shared/components/mutationTable/column/ExonColu
 
 export interface IPatientViewMutationTableProps extends IMutationTableProps {
     sampleManager:SampleManager | null;
+    sampleToGenePanelId:{[sampleId: string]: string|undefined};
```

Comment:
so undefined indicates either not profiled or whole exome? or just whole exome?

Reply:
An `undefined` value for genePanelId means that the sample  was analyzed with a whole genome/exome approach.